node/blockchain/inner_models/signed_change_request_message/pv_schedule_update.py

```diff
@@ -30,8 +30,8 @@ def validate_nodes_are_declared(self, blockchain_facade):
                 raise ValidationError('All nodes in the schedule must be declared')
 
     def validate_block_numbers(self, blockchain_facade):
-        if min(int(key) for key in self.schedule.keys()) < blockchain_facade.get_next_block_number():
-            raise ValidationError('Schedule keys must be equal or more than next block number')
+        if min(int(key) for key in self.schedule.keys()) != blockchain_facade.get_next_block_number():
```

Comment:
this is wrong as well as previous implementation. Image we had this schedule for next block number = 103:
```
{
"100": ID1,
"200": ID2
}
```

Now PV wants to add another node to the schedule. It must issue this signed change request (next block number will be incremented to 104 after making block out of it:
```
{
"100": ID1,
"200": ID2,
"300": ID3
}
```
And this is perfectly valid schedule.

While this one would be not valid, since block range from 0 to 99 had already passed:
```
{
"0": ID0,
"100": ID1,
"200": ID2,
"300": ID3
}
```

This one is not valid too, since there is no PV to validate block 104:
```
{
"300": ID3
}
```